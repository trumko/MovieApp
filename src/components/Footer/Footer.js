import React from "react";
import Logo from 'components/Logo/Logo';

// import './Footer.scss'

export default function Footer() {
    return (
      <div className="mr_footer">
        <div className="mr_container">
          <Logo />
        </div>
      </div>
    )
}
