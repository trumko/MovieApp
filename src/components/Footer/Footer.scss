@import "styles/variables.scss";

.mr_footer {
  background: $black;
}
