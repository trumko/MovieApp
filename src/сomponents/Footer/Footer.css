.mr_footer {
  background: black;
  color: #c13a3a;
  font-family: sans-serif;
  font-weight: bold;
}


.mr_testContent {
  float: right;
  color: #fff;
}


.mr_testContent a {
  color: #fff;
}
