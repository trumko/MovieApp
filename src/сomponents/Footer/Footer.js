import React, { Component } from "react";
import { Link } from 'react-router-dom';

import './Footer.css'

export default class Footer extends Component {
  render() {
    return (
      <div className="mr_footer">
        <div className="mr_container">
          <span>netflixroulette</span>
          <span className="mr_testContent">
            <Link to="/">Home page</Link>
            <span> | </span>
            <Link to="/test">Test page</Link>
            <span> | </span>
            <Link to="/movie/1">Movie page</Link>
          </span>
        </div>
      </div>
    )
  }
}
