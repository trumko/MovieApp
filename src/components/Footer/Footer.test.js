import React from 'react'
import { shallow } from 'enzyme'
import Footer from './Footer'

describe('Footer', () => {
  it('should be rendered correctly', () => {
    const component = shallow(<Footer />);

    expect(component).toMatchSnapshot();
  });
});
