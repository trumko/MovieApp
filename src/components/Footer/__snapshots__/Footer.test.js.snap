// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should be rendered correctly 1`] = `
<div
  className="mr_footer"
>
  <div
    className="mr_container"
  >
    <Logo />
  </div>
</div>
`;
